flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -65,7 +65,11 @@
 		key("yarn.containers.vcores")
 		.defaultValue(-1)
 		.withDescription("The number of virtual cores (vcores) per YARN container. By default, the number of vcores" +
-			" is set to the number of slots per TaskManager, if set, or to 1, otherwise.");
+			" is set to the number of slots per TaskManager, if set, or to 1, otherwise. In order for this parameter " +
+			"to be used your cluster must have CPU scheduling enabled. You can do this e.g. by setting the " +
+			"org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler or enabling " +
+			"org.apache.hadoop.yarn.util.resource.DominantResourceCalculator for " +
+			"org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler");
```

Comment:
Have you actually tried - back when I created this issue, simply changing to `DominantResourceCalculator` did not lead to the desired effect.

Reply:
Also, please put the configuration parameters into some code environment, e.g. `<tt>org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler</tt>`, for the config.html page.

Reply:
You were right @NicoK . The `DominantResourceCalculator` didn't work for me in 2.8.3. Sorry for the confusion.